db/fixtures/customization_templates.yml

```diff
@@ -299,12 +299,17 @@
     # line tool. Note that this expects the subnet mask in the prefix format,
     # for example 24 instead of 255.255.255.0.
     runcmd:
-    - nmcli connection down eth0
-    - nmcli connection modify eth0
+    - ETH_CON=$(nmcli --get-values name connection show | grep "eth")
+    - nmcli connection down "$ETH_CON"
+    - nmcli connection modify "$ETH_CON"
       ip4 <%= evm[:ip_addr] %>/<%= evm[:subnet_mask] %>
```

Comment:
Do we want to avoid setting the rest of these `ip4` attributes if it's marked as dhcp?

Reply:
@bdunne I also thought about this but I am not sure. Because you might want to assign static IP to to a VM and still want to get one from DHCP. Those two options actually work together if you go for Address Mode == DHCP.

Reply:
Currently in our dialogs these fields are only visible if you select "static", right?

Reply:
@bdunne No, they are visible no matter which Address Mode is selected. If they were visible only with "static" Address Mode selected, then your original proposal would be without doubt the right way to go.